install.sh

```diff
@@ -994,11 +994,27 @@ case "${SHELL}" in
     ;;
 esac
 
-cat <<EOS
+in_shell_profile=$(grep -qs "eval \"\$(${HOMEBREW_PREFIX}/bin/brew shellenv)\"" "${shell_profile}")
+
+# FIXME: figure out why this give a "command not found" error
+# show different instructions on adding Homebrew to path based on if the user has already done so
+if "${in_shell_profile}" && ! type brew &>/dev/null
```

Comment:
```suggestion
if [[ -n $in_shell_profile ]] && ! type brew &>/dev/null
```
This requires the `grep` fix I mentioned above.